File: C  atau C++/latihan2.cpp
#include <iostream>
using namespace std;

int main() {
    const float PI = 3.14159;  // Konstanta PI
    float r;                   // Variabel jari-jari

    cout << "Masukkan jari-jari: ";
    cin >> r;

    float luas = PI * r * r;   // Rumus luas lingkaran
    cout << "Luas lingkaran = " << luas;

    return 0;
}

File: php/latihan2.php
<?php
// Mendefinisikan konstanta PI
define("PI", 3.14159);

// Membaca input jari-jari dari pengguna
$r = readline("Masukkan jari-jari: ");

// Menghitung luas lingkaran
$luas = PI * $r * $r;

// Menampilkan hasil
echo "Luas lingkaran = $luas";
?>